_partials/_header.jade

```diff
@@ -3,4 +3,24 @@ header(class='row main-cta')
         h2 #{ header_text }
         p #{ subheader_text }
         // "Get involved", "Become a Hacker"
-        a(class='button large', href='get-started-with-civic-hacking') Get Involved
\ No newline at end of file
+
+        section.row.block
+          .small-12.medium-8.columns
+            h1 Submit your ideas for making Miami great!
+        section(class="row block")
+          div(class="small-12 medium-8 large-5 columns")
+            form(action='https://formspree.io/codeformiami@gmail.com', method='POST', autocomplete='off')
+              label(for='FullName') Full name
+              input(type='text', name='name', id='FullName', placeholder='Type your name here', autocomplete='off', required)
+              label(for='IdeaMessage') Your idea
+              textarea(name='message', id='IdeaMessage', placeholder='My idea is...', autocomplete='off', required)
+              button(type='submit') Send
+              input(type='hidden', name='_subject', value='Miami Improvement Idea', autocomplete='off')
+              input(type='hidden', name='_format', value='plain', autocomplete='off')
+              input(type='hidden', name='_next', value='http://codefor.miami/ideas-thanks', autocomplete='off')
+              script(type='text/javascript').
```

Comment:
add the div that says 'Any issues with this form?' like in ideas.jade, you can copy paste the lines